frappe/integrations/doctype/paypal_settings/paypal_settings.py

```diff
@@ -387,7 +387,7 @@ def ipn_handler():
 		doc = frappe.get_doc({
```

Comment:
this is a callback for a remote request, let it reflect that using `is_remote_request`